plugins/inputs/knx_listener/knx_listener.go

```diff
@@ -195,5 +195,5 @@ func (kl *KNXListener) listen() {
 }
 
 func init() {
-	inputs.Add("KNXListener", func() telegraf.Input { return &KNXListener{ServiceType: "tunnel"} })
```

Comment:
knx_listener is the right name to use, but since we released this in 1.19 with KNXListener, we should register both names here for backward compatibility.